docs/examples/nginx/templates/deployment.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  # This uses a "fullname" template (see _helpers)
+  # Basing names on .Release.Name means that the same chart can be installed
+  # multiple times into the same namespace.
+  name: {{template "fullname" .}}
+  labels:
+    heritage: helm
+    # This makes it easy to search using kubectl
+    release: {{.Release.Name}}
+    # This makes it easy to audit chart usage.
+    chart: {{.Chart.Name}}-{{.Chart.Version}}
+spec:
+  replicas: {{default 1 .replicaCount}}
+  template:
+    metadata:
+      labels:
+        app: {{template "fullname" .}}
+        release: {{.Release.Name}}
```

Comment:
Naturally, a use will likely read `docs/examples/alpine` first, and the [metadata there](https://github.com/kubernetes/helm/blob/0acbcdd5ccaceedf26871b416e2d775bf6e1c026/docs/examples/alpine/templates/alpine-pod.yaml#L3-L10) differs from this. Would be great to clarify what user really must supply.